guides/codebase/naming/mapper.md

```diff
@@ -0,0 +1,23 @@
+# Mapper Naming Convention(s)
+
+Keep in mind that **all mappers have to be based off of an existing Entity (./src/entities) in the database.** This means all files in the mappers folder have to be named after an existing Entity (like {Entity}Mapper.ts).
+
+## Optional Default Mapper Functions
+
+For all files in ./src/mappers, if you want, you can have these three default mapper function names. They are all optional, and you can have any of the three in your mapper class (no need to have all three of them). The three default function names are:
+
+- `requestToNewEntity`
```

Comment:
we might have to revisit this later but thanks for putting it in anyways

Reply:
Yep np, I also think this is fine for now but we'll probably end up amending this at some point.